docs/cashmere-import.scss

```diff
@@ -0,0 +1 @@
+@import "../lib/src/colors.scss";
```

Comment:
It took me a while to realize that this file is just making it easier for other files to import. I like it, but I think a comment would be needed to explain the single import